packages/contracts/deploy/00_managing-dao/00_managing-dao.ts

```diff
@@ -23,7 +23,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       execute: {
         init: {
           methodName: 'initialize',
-          args: ['0x00', deployer, ethers.constants.AddressZero],
+          args: ['0x00', deployer, ethers.constants.AddressZero, '0x'],
```

Comment:
add comment here that says what this is, mark it with TODO: so we know we might need to update it.

Reply:
converted to params so it won't need comments.